src/libraries/wizard/componentselector.h

```diff
@@ -26,8 +26,8 @@
 #pragma once
 
 #include <QModelIndex>
-#include <QStringList>
 #include <QTreeView>
+#include <utils/fileutils.h>
```

Comment:
shouldn't there be an empty line to separate include groups? again - I do not remember the convention exactly

Reply:
i executed clang-format again
everything is ok